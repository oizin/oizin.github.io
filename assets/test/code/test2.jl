# This file was generated, do not modify it. # hide
df = dataset("plm", "Gasoline")
println(df)