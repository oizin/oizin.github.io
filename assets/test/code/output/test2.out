342×6 DataFrame
 Row │ Country   Year   LGasPCar  LIncomeP  LRPMG       LCarPCap
     │ Cat…      Int32  Float64   Float64   Float64     Float64
─────┼────────────────────────────────────────────────────────────
   1 │ AUSTRIA    1960   4.17324  -6.47428  -0.334548    -9.76684
   2 │ AUSTRIA    1961   4.10099  -6.42601  -0.351328    -9.60862
   3 │ AUSTRIA    1962   4.07318  -6.40731  -0.379518    -9.45726
   4 │ AUSTRIA    1963   4.05951  -6.37068  -0.414251    -9.34315
   5 │ AUSTRIA    1964   4.03769  -6.32225  -0.445335    -9.23774
   6 │ AUSTRIA    1965   4.03398  -6.29467  -0.497061    -9.1239
   7 │ AUSTRIA    1966   4.04754  -6.25255  -0.466838    -9.01982
   8 │ AUSTRIA    1967   4.05291  -6.23458  -0.505883    -8.9344
   9 │ AUSTRIA    1968   4.04551  -6.20689  -0.522413    -8.84797
  10 │ AUSTRIA    1969   4.04635  -6.15314  -0.559111    -8.78869
  11 │ AUSTRIA    1970   4.08089  -6.08171  -0.596561    -8.7282
  12 │ AUSTRIA    1971   4.10672  -6.04363  -0.654459    -8.6359
  13 │ AUSTRIA    1972   4.12802  -5.98105  -0.596332    -8.53834
  14 │ AUSTRIA    1973   4.19938  -5.89515  -0.594447    -8.48729
  15 │ AUSTRIA    1974   4.0185   -5.85238  -0.466027    -8.4304
  16 │ AUSTRIA    1975   4.02902  -5.86936  -0.454142    -8.38281
  17 │ AUSTRIA    1976   3.98541  -5.8117   -0.500084    -8.32223
  18 │ AUSTRIA    1977   3.93168  -5.83329  -0.421916    -8.24956
  19 │ AUSTRIA    1978   3.92275  -5.76202  -0.469603    -8.21104
  20 │ BELGIUM    1960   4.16402  -6.21509  -0.16571     -9.40553
  21 │ BELGIUM    1961   4.12436  -6.17684  -0.171731    -9.30315
  22 │ BELGIUM    1962   4.07596  -6.12964  -0.222291    -9.21807
  23 │ BELGIUM    1963   4.00127  -6.09402  -0.250462    -9.11493
  24 │ BELGIUM    1964   3.99438  -6.03646  -0.275911    -9.00549
  25 │ BELGIUM    1965   3.95153  -6.00725  -0.344937    -8.86258
  26 │ BELGIUM    1966   3.82054  -5.99411  -0.236398    -8.75453
  27 │ BELGIUM    1967   3.90688  -5.96481  -0.266995    -8.74883
  28 │ BELGIUM    1968   3.82867  -5.92469  -0.311161    -8.57651
  29 │ BELGIUM    1969   3.8546   -5.85753  -0.354809    -8.52145
  30 │ BELGIUM    1970   3.87039  -5.7972   -0.37794     -8.45304
  31 │ BELGIUM    1971   3.87225  -5.76105  -0.39923     -8.40946
  32 │ BELGIUM    1972   3.9054   -5.71023  -0.310646    -8.36259
  33 │ BELGIUM    1973   3.896    -5.64414  -0.373092    -8.31445
  34 │ BELGIUM    1974   3.81823  -5.60073  -0.362236    -8.2706
  35 │ BELGIUM    1975   3.87778  -5.623    -0.364308    -8.22885
  36 │ BELGIUM    1976   3.86415  -5.56719  -0.378966    -8.18413
  37 │ BELGIUM    1977   3.85431  -5.5567   -0.431641    -8.13853
  38 │ BELGIUM    1978   3.84274  -5.53294  -0.59095     -8.10474
  39 │ CANADA     1960   4.85524  -5.88971  -0.972106    -8.37892
  40 │ CANADA     1961   4.82656  -5.88434  -0.97229     -8.34673
  41 │ CANADA     1962   4.85053  -5.84455  -0.978608    -8.32051
  42 │ CANADA     1963   4.83808  -5.79235  -1.01905     -8.26942
  43 │ CANADA     1964   4.83976  -5.76006  -1.00286     -8.2524
  44 │ CANADA     1965   4.85083  -5.72282  -1.01713     -8.22362
  45 │ CANADA     1966   4.87102  -5.67178  -1.01694     -8.20124
  46 │ CANADA     1967   4.8525   -5.60848  -1.0236      -8.15608
  47 │ CANADA     1968   4.86878  -5.57392  -1.01985     -8.12319
  48 │ CANADA     1969   4.86443  -5.56085  -1.03686     -8.09511
  49 │ CANADA     1970   4.89957  -5.51476  -1.06733     -8.08003
  50 │ CANADA     1971   4.89507  -5.47992  -1.05804     -8.03926
  51 │ CANADA     1972   4.8893   -5.4366   -1.09967     -7.98953
  52 │ CANADA     1973   4.89969  -5.41475  -1.13316     -7.94214
  53 │ CANADA     1974   4.89159  -5.41846  -1.1238      -7.90076
  54 │ CANADA     1975   4.88847  -5.3791   -1.18568     -7.87331
  55 │ CANADA     1976   4.83736  -5.36129  -1.0618      -7.80842
  56 │ CANADA     1977   4.81099  -5.33697  -1.07084     -7.76879
  57 │ CANADA     1978   4.85585  -5.31127  -1.07495     -7.78806
  58 │ DENMARK    1960   4.50199  -6.06173  -0.195703    -9.32616
  59 │ DENMARK    1961   4.48285  -6.0009   -0.253618    -9.19314
  60 │ DENMARK    1962   4.38545  -5.98748  -0.218754    -9.04728
  61 │ DENMARK    1963   4.354    -5.97306  -0.248009    -8.95278
  62 │ DENMARK    1964   4.32644  -5.89474  -0.306549    -8.85261
  63 │ DENMARK    1965   4.24945  -5.85936  -0.327015    -8.76195
  64 │ DENMARK    1966   4.23364  -5.85187  -0.396188    -8.68154
  65 │ DENMARK    1967   4.20347  -5.82306  -0.442574    -8.60212
  66 │ DENMARK    1968   4.16169  -5.79226  -0.352048    -8.53377
  67 │ DENMARK    1969   4.17356  -5.72277  -0.406879    -8.47046
  68 │ DENMARK    1970   4.12881  -5.70831  -0.440461    -8.42706
  69 │ DENMARK    1971   4.10315  -5.67682  -0.45474     -8.3694
  70 │ DENMARK    1972   4.08281  -5.64599  -0.499189    -8.33467
  71 │ DENMARK    1973   4.13744  -5.59586  -0.432572    -8.3046
  72 │ DENMARK    1974   4.00046  -5.59487  -0.425177    -8.30018
  73 │ DENMARK    1975   4.03301  -5.61297  -0.393954    -8.27463
  74 │ DENMARK    1976   4.00774  -5.55394  -0.353615    -8.24409
  75 │ DENMARK    1977   4.00529  -5.53873  -0.356909    -8.21986
  76 │ DENMARK    1978   4.0366   -5.48307  -0.290681    -8.19581
  77 │ FRANCE     1960   3.9077   -6.26436  -0.0195983   -9.14571
  78 │ FRANCE     1961   3.88562  -6.22088  -0.02386     -9.04425
  79 │ FRANCE     1962   3.82367  -6.17359  -0.0689202   -8.93013
  80 │ FRANCE     1963   3.789    -6.13706  -0.137929    -8.81863
  81 │ FRANCE     1964   3.76708  -6.08724  -0.197846    -8.71096
  82 │ FRANCE     1965   3.76058  -6.04853  -0.233653    -8.62939
  83 │ FRANCE     1966   3.74953  -6.00084  -0.264272    -8.54577
  84 │ FRANCE     1967   3.76862  -5.95566  -0.294058    -8.48737
  85 │ FRANCE     1968   3.77823  -5.90138  -0.323162    -8.42555
  86 │ FRANCE     1969   3.77346  -5.84077  -0.315191    -8.36914
  87 │ FRANCE     1970   3.80158  -5.78418  -0.333846    -8.32709
  88 │ FRANCE     1971   3.82596  -5.74314  -0.379457    -8.27952
  89 │ FRANCE     1972   3.84668  -5.69911  -0.407816    -8.22276
  90 │ FRANCE     1973   3.88499  -5.65507  -0.475034    -8.1839
  91 │ FRANCE     1974   3.80799  -5.6301   -0.216982    -8.16033
  92 │ FRANCE     1975   3.80855  -5.63327  -0.258382    -8.14547
  93 │ FRANCE     1976   3.90812  -5.58834  -0.246513    -8.11041
  94 │ FRANCE     1977   3.8125   -5.56342  -0.225507    -8.06415
  95 │ FRANCE     1978   3.78888  -5.53025  -0.380759    -8.00565
  96 │ GERMANY    1960   3.91695  -6.15984  -0.185911    -9.34248
  97 │ GERMANY    1961   3.88535  -6.12092  -0.230954    -9.18384
  98 │ GERMANY    1962   3.87148  -6.09426  -0.343842    -9.03728
  99 │ GERMANY    1963   3.84878  -6.06836  -0.374647    -8.91363
 100 │ GERMANY    1964   3.86899  -6.01344  -0.399653    -8.81101
 101 │ GERMANY    1965   3.86105  -5.96647  -0.439878    -8.71189
 102 │ GERMANY    1966   3.88074  -5.94907  -0.540002    -8.63119
 103 │ GERMANY    1967   3.87503  -5.96004  -0.549981    -8.57601
 104 │ GERMANY    1968   3.88936  -5.88341  -0.438242    -8.51586
 105 │ GERMANY    1969   3.89918  -5.82964  -0.589231    -8.43806
 106 │ GERMANY    1970   3.90252  -5.76737  -0.633295    -8.36193
 107 │ GERMANY    1971   3.9321   -5.73802  -0.671763    -8.28382
 108 │ GERMANY    1972   3.9324   -5.71875  -0.717975    -8.23716
 109 │ GERMANY    1973   3.92416  -5.66817  -0.725875    -8.199
 110 │ GERMANY    1974   3.88821  -5.66396  -0.569829    -8.18268
 111 │ GERMANY    1975   3.92203  -5.67906  -0.564824    -8.14734
 112 │ GERMANY    1976   3.89647  -5.62384  -0.624813    -8.08706
 113 │ GERMANY    1977   3.89569  -5.59461  -0.597612    -8.0111
 114 │ GERMANY    1978   3.88388  -5.56173  -0.628173    -7.95008
 115 │ GREECE     1960   5.03741  -7.16486  -0.0835474  -12.1738
 116 │ GREECE     1961   5.38149  -7.07335  -0.10422    -12.0517
 117 │ GREECE     1962   5.18807  -7.05142  -0.133208   -11.9064
 118 │ GREECE     1963   5.08341  -6.98202  -0.156536   -11.7337
 119 │ GREECE     1964   5.17863  -6.89871  -0.180518   -11.55
 120 │ GREECE     1965   5.01511  -6.82293  -0.07794    -11.3171
 121 │ GREECE     1966   5.0484   -6.7751   -0.114919   -11.1567
 122 │ GREECE     1967   4.97578  -6.73243  -0.137758   -11.0109
 123 │ GREECE     1968   4.94416  -6.67681  -0.153759   -10.8476
 124 │ GREECE     1969   4.89477  -6.5911   -0.17987    -10.7138
 125 │ GREECE     1970   4.85182  -6.51027  -0.202524   -10.5645
 126 │ GREECE     1971   4.81181  -6.43745  -0.0676108  -10.4177
 127 │ GREECE     1972   4.79571  -6.34804  -0.119731   -10.2861
 128 │ GREECE     1973   4.76858  -6.28117  -0.0519103  -10.1565
 129 │ GREECE     1974   4.62781  -6.32458   0.316254   -10.0717
 130 │ GREECE     1975   4.58096  -6.27159   0.206316    -9.93333
 131 │ GREECE     1976   4.47996  -6.22476   0.193193    -9.77696
 132 │ GREECE     1977   4.48462  -6.20337   0.23503     -9.6157
 133 │ GREECE     1978   4.5464   -6.15112   0.16896     -9.57498
 134 │ IRELAND    1960   4.27042  -6.72247  -0.0764812   -9.69814
 135 │ IRELAND    1961   4.25524  -6.65817  -0.120409    -9.60487
 136 │ IRELAND    1962   4.20843  -6.62281  -0.1416      -9.50216
 137 │ IRELAND    1963   4.1767   -6.59117  -0.152329    -9.40933
 138 │ IRELAND    1964   4.18624  -6.56428  -0.244282    -9.31442
 139 │ IRELAND    1965   4.1649   -6.5483   -0.168994    -9.21942
 140 │ IRELAND    1966   4.16821  -6.55196  -0.210719    -9.17091
 141 │ IRELAND    1967   4.18899  -6.50048  -0.173835    -9.11469
 142 │ IRELAND    1968   4.18112  -6.60796  -0.213393    -9.0172
 143 │ IRELAND    1969   4.20861  -6.37974  -0.271628    -8.94727
 144 │ IRELAND    1970   4.24994  -6.36652  -0.32069     -8.91893
 145 │ IRELAND    1971   4.26712  -6.34544  -0.360411    -8.87195
 146 │ IRELAND    1972   4.27077  -6.31357  -0.423931    -8.82395
 147 │ IRELAND    1973   4.25383  -6.29658  -0.645673    -8.75577
 148 │ IRELAND    1974   4.32559  -6.28688  -0.553439    -8.7452
 149 │ IRELAND    1975   4.24822  -6.30196  -0.641264    -8.70948
 150 │ IRELAND    1976   4.22115  -6.28727  -0.661343    -8.66722
 151 │ IRELAND    1977   4.24155  -6.25113  -0.560115    -8.64073
 152 │ IRELAND    1978   4.19861  -6.19314  -0.662778    -8.55099
 153 │ ITALY      1960   4.05073  -6.72749   0.165077   -10.1421
 154 │ ITALY      1961   4.04525  -6.65535  -0.0855904   -9.93266
 155 │ ITALY      1962   4.029    -6.59978  -0.183513    -9.7267
 156 │ ITALY      1963   3.96309  -6.55267  -0.265414    -9.4788
 157 │ ITALY      1964   3.93632  -6.6293   -0.426096    -9.30847
 158 │ ITALY      1965   3.87499  -6.60219  -0.327126    -9.15802
 159 │ ITALY      1966   3.82274  -6.44968  -0.248874    -9.01481
 160 │ ITALY      1967   3.78581  -6.3935   -0.1916      -8.88328
 161 │ ITALY      1968   3.7639   -6.33451  -0.206167    -8.7642
 162 │ ITALY      1969   3.73739  -6.28286  -0.247567    -8.666
 163 │ ITALY      1970   3.70687  -6.24045  -0.232715    -8.56734
 164 │ ITALY      1971   3.64821  -6.2226   -0.148223    -8.47097
 165 │ ITALY      1972   3.62924  -6.20565  -0.215089    -8.38018
 166 │ ITALY      1973   3.65226  -6.1573   -0.325085    -8.3167
 167 │ ITALY      1974   3.49947  -6.12584  -0.222909    -8.26226
 168 │ ITALY      1975   3.51568  -6.1701   -0.0327091   -8.21767
 169 │ ITALY      1976   3.42763  -6.11922   0.102928    -8.16776
 170 │ ITALY      1977   3.38021  -6.10454   0.164188    -8.14566
 171 │ ITALY      1978   3.3945   -6.08369   0.0348221   -8.11285
 172 │ JAPAN      1960   5.99529  -6.9862   -0.145323   -12.2351
 173 │ JAPAN      1961   5.75848  -6.89295  -0.148749   -11.8705
 174 │ JAPAN      1962   5.64192  -6.80583  -0.187315   -11.5879
 175 │ JAPAN      1963   5.50808  -6.72276  -0.199965   -11.2702
 176 │ JAPAN      1964   5.27533  -6.63592  -0.203864   -10.977
 177 │ JAPAN      1965   5.12585  -6.58556  -0.237866   -10.722
 178 │ JAPAN      1966   5.00082  -6.50019  -0.274115   -10.4695
 179 │ JAPAN      1967   4.85486  -6.3776   -0.331672   -10.1767
 180 │ JAPAN      1968   4.68467  -6.25188  -0.351269    -9.88185
 181 │ JAPAN      1969   4.51829  -6.15931  -0.41685     -9.6076
 182 │ JAPAN      1970   4.4016   -6.05874  -0.462035    -9.38295
 183 │ JAPAN      1971   4.30562  -6.00608  -0.439414    -9.20939
 184 │ JAPAN      1972   4.22298  -5.93469  -0.521001    -9.05414
 185 │ JAPAN      1973   4.09388  -5.83587  -0.462707    -8.92427
 186 │ JAPAN      1974   4.00796  -5.85255  -0.190906    -8.84652
 187 │ JAPAN      1975   3.99598  -5.85171  -0.159485    -8.77518
 188 │ JAPAN      1976   3.98816  -5.80043  -0.207266    -8.71642
 189 │ JAPAN      1977   3.96467  -5.75686  -0.219044    -8.65553
 190 │ JAPAN      1978   3.94875  -5.70884  -0.287076    -8.59403
 191 │ NETHERLA   1960   4.64627  -6.21636  -0.201485    -9.99845
 192 │ NETHERLA   1961   4.55984  -6.20816  -0.215993    -9.84671
 193 │ NETHERLA   1962   4.47172  -6.18061  -0.25968     -9.69057
 194 │ NETHERLA   1963   4.37769  -6.16256  -0.297187    -9.53403
 195 │ NETHERLA   1964   4.31956  -6.08991  -0.369294    -9.34611
 196 │ NETHERLA   1965   4.233    -6.05126  -0.341975    -9.17516
 197 │ NETHERLA   1966   4.16802  -6.03886  -0.34809     -9.02348
 198 │ NETHERLA   1967   4.11915  -5.99972  -0.31232     -8.89623
 199 │ NETHERLA   1968   4.07073  -5.94662  -0.444504    -8.75857
 200 │ NETHERLA   1969   3.98769  -5.88056  -0.41695     -8.6341
 201 │ NETHERLA   1970   3.95641  -5.83858  -0.399545    -8.5195
 202 │ NETHERLA   1971   3.9478   -5.81304  -0.43393     -8.45759
 203 │ NETHERLA   1972   3.9153   -5.78545  -0.319032    -8.38263
 204 │ NETHERLA   1973   3.88048  -5.74729  -0.427282    -8.30671
 205 │ NETHERLA   1974   3.71138  -5.72925  -0.352537    -8.25049
 206 │ NETHERLA   1975   3.71847  -5.74301  -0.434262    -8.1833
 207 │ NETHERLA   1976   3.78363  -5.70387  -0.429084    -8.20247
 208 │ NETHERLA   1977   3.77668  -5.68673  -0.464742    -8.16254
 209 │ NETHERLA   1978   3.88259  -5.66066  -0.557915    -8.15601
 210 │ NORWAY     1960   4.43504  -6.09036  -0.13969     -9.67505
 211 │ NORWAY     1961   4.33272  -6.03716  -0.157905    -9.4716
 212 │ NORWAY     1962   4.23424  -6.0129   -0.199088    -9.32957
 213 │ NORWAY     1963   4.21619  -5.96649  -0.232633    -9.21773
 214 │ NORWAY     1964   4.16435  -5.92791  -0.263747    -9.09154
 215 │ NORWAY     1965   4.13163  -5.88157  -0.315931    -8.988
 216 │ NORWAY     1966   4.08461  -5.85122  -0.250117    -8.89196
 217 │ NORWAY     1967   4.06431  -5.80556  -0.265558    -8.80268
 218 │ NORWAY     1968   4.10386  -5.77349  -0.300368    -8.80113
 219 │ NORWAY     1969   4.08682  -5.73532  -0.33823     -8.69459
 220 │ NORWAY     1970   4.08881  -5.74114  -0.390726    -8.62862
 221 │ NORWAY     1971   4.07893  -5.69944  -0.301272    -8.56849
 222 │ NORWAY     1972   4.07726  -5.65801  -0.260239    -8.51439
 223 │ NORWAY     1973   4.06135  -5.62317  -0.338808    -8.45966
 224 │ NORWAY     1974   3.96033  -5.57838  -0.151009    -8.40644
 225 │ NORWAY     1975   3.99602  -5.55189  -0.327268    -8.34289
 226 │ NORWAY     1976   3.97821  -5.49764  -0.353088    -8.27729
 227 │ NORWAY     1977   3.98852  -5.45807  -0.382558    -8.20284
 228 │ NORWAY     1978   4.00249  -5.42329  -0.307659    -8.17179
 229 │ SPAIN      1960   4.74941  -6.16609   1.12531    -11.5884
 230 │ SPAIN      1961   4.58924  -6.05779   1.10956    -11.384
 231 │ SPAIN      1962   4.42909  -5.98048   1.057      -11.1578
 232 │ SPAIN      1963   4.3465   -5.90512   0.976835   -10.9845
 233 │ SPAIN      1964   4.30058  -5.85853   0.915323   -10.7879
 234 │ SPAIN      1965   4.21721  -5.80065   0.816661   -10.5852
 235 │ SPAIN      1966   4.11378  -5.7351    0.756718   -10.3297
 236 │ SPAIN      1967   4.08632  -5.70559   0.741308   -10.103
 237 │ SPAIN      1968   4.04425  -5.65733   0.703865    -9.91147
 238 │ SPAIN      1969   3.9941   -5.60105   0.66949     -9.72043
 239 │ SPAIN      1970   3.96605  -5.55325   0.612172    -9.55736
 240 │ SPAIN      1971   3.90734  -5.51554   0.606996    -9.40996
 241 │ SPAIN      1972   3.89122  -5.45871   0.537168    -9.2704
 242 │ SPAIN      1973   3.91281  -5.37372   0.433772    -9.12663
 243 │ SPAIN      1974   3.76857  -5.3296    0.524921    -9.0133
 244 │ SPAIN      1975   3.75245  -5.33093   0.629555    -8.91602
 245 │ SPAIN      1976   3.71087  -5.31285   0.683854    -8.82064
 246 │ SPAIN      1977   3.65073  -5.29909   0.526272    -8.72722
 247 │ SPAIN      1978   3.62044  -5.28596   0.621414    -8.6347
 248 │ SWEDEN     1960   4.06301  -8.07252  -2.52042     -8.74268
 249 │ SWEDEN     1961   4.06187  -8.01959  -2.57148     -8.65988
 250 │ SWEDEN     1962   4.00644  -7.99725  -2.53448     -8.57742
 251 │ SWEDEN     1963   4.00277  -7.96675  -2.60511     -8.49431
 252 │ SWEDEN     1964   4.02486  -7.89758  -2.65802     -8.43347
 253 │ SWEDEN     1965   4.01546  -7.87481  -2.64477     -8.36949
 254 │ SWEDEN     1966   4.00256  -7.8608   -2.63901     -8.32686
 255 │ SWEDEN     1967   3.99735  -7.84546  -2.6561      -8.28948
 256 │ SWEDEN     1968   3.99758  -7.81399  -2.67919     -8.24823
 257 │ SWEDEN     1969   3.99172  -7.77108  -2.7319      -8.19746
 258 │ SWEDEN     1970   3.98937  -7.73261  -2.73359     -8.16451
 259 │ SWEDEN     1971   3.98206  -7.76011  -2.77885     -8.14223
 260 │ SWEDEN     1972   3.98007  -7.74251  -2.77468     -8.10314
 261 │ SWEDEN     1973   4.03056  -7.7253   -2.84143     -8.08706
 262 │ SWEDEN     1974   3.91316  -7.68953  -2.79841     -8.0366
 263 │ SWEDEN     1975   3.97384  -7.67956  -2.76731     -7.99422
 264 │ SWEDEN     1976   3.984    -7.67204  -2.82294     -7.95607
 265 │ SWEDEN     1977   4.03101  -7.70738  -2.82006     -7.9688
 266 │ SWEDEN     1978   4.06737  -7.6792   -2.8965      -7.97193
 267 │ SWITZERL   1960   4.39762  -6.15607  -0.823218    -9.2624
 268 │ SWITZERL   1961   4.44133  -6.11164  -0.865585    -9.15823
 269 │ SWITZERL   1962   4.28715  -6.09301  -0.822185    -9.04614
 270 │ SWITZERL   1963   4.31248  -6.06802  -0.86012     -8.95078
 271 │ SWITZERL   1964   4.31343  -6.02147  -0.867677    -8.83943
 272 │ SWITZERL   1965   4.28871  -5.99609  -0.905287    -8.76037
 273 │ SWITZERL   1966   4.26635  -5.98162  -0.859567    -8.67912
 274 │ SWITZERL   1967   4.26592  -5.97492  -0.906567    -8.61996
 275 │ SWITZERL   1968   4.21779  -5.95363  -0.872325    -8.54475
 276 │ SWITZERL   1969   4.21129  -5.91217  -0.918122    -8.47338
 277 │ SWITZERL   1970   4.2253   -5.87864  -0.963442    -8.40644
 278 │ SWITZERL   1971   4.2807   -5.84867  -1.03746     -8.36007
 279 │ SWITZERL   1972   4.25916  -5.81149  -0.940153    -8.30363
 280 │ SWITZERL   1973   4.22982  -5.76041  -0.867228    -8.24664
 281 │ SWITZERL   1974   4.16636  -5.7517   -0.886923    -8.20551
 282 │ SWITZERL   1975   4.12057  -5.82578  -0.884758    -8.16608
 283 │ SWITZERL   1976   4.07957  -5.83596  -0.907362    -8.12875
 284 │ SWITZERL   1977   4.10053  -5.82096  -0.911473    -8.09351
 285 │ SWITZERL   1978   4.05005  -5.81685  -1.03209     -8.03437
 286 │ TURKEY     1960   6.12955  -7.80114  -0.253408   -13.4752
 287 │ TURKEY     1961   6.10621  -7.78673  -0.342524   -13.3847
 288 │ TURKEY     1962   6.08459  -7.83627  -0.408205   -13.2459
 289 │ TURKEY     1963   6.07513  -7.63119  -0.224992   -13.255
 290 │ TURKEY     1964   6.0646   -7.6269   -0.252194   -13.2103
 291 │ TURKEY     1965   5.82305  -7.62203  -0.293476   -12.8793
 292 │ TURKEY     1966   6.15664  -7.5109   -0.356405   -12.9511
 293 │ TURKEY     1967   6.04448  -7.46081  -0.33515    -12.8019
 294 │ TURKEY     1968   6.07659  -7.4212   -0.365074   -12.8074
 295 │ TURKEY     1969   5.72071  -7.38865  -0.298454   -12.5185
 296 │ TURKEY     1970   5.72211  -7.32409  -0.398826   -12.4466
 297 │ TURKEY     1971   5.66965  -7.18237  -0.304619   -12.3988
 298 │ TURKEY     1972   5.57882  -7.08372  -0.546374   -12.192
 299 │ TURKEY     1973   5.68647  -7.06742  -0.69162    -11.9902
 300 │ TURKEY     1974   5.42773  -7.01281  -0.339653   -11.7594
 301 │ TURKEY     1975   5.42678  -6.95444  -0.537947   -11.555
 302 │ TURKEY     1976   5.31293  -6.9097   -0.75141    -11.4316
 303 │ TURKEY     1977   5.31346  -6.89366  -0.955524   -11.2338
 304 │ TURKEY     1978   5.14126  -6.88883  -0.35291    -11.1813
 305 │ U.K.       1960   4.10024  -6.18685  -0.391086    -9.11762
 306 │ U.K.       1961   4.08864  -6.16886  -0.451853    -9.04889
 307 │ U.K.       1962   4.04805  -6.16672  -0.422877    -8.96688
 308 │ U.K.       1963   3.98527  -6.13067  -0.463351    -8.85594
 309 │ U.K.       1964   3.97678  -6.08637  -0.495774    -8.74978
 310 │ U.K.       1965   3.96868  -6.07689  -0.426549    -8.67761
 311 │ U.K.       1966   3.95721  -6.06906  -0.470681    -8.61828
 312 │ U.K.       1967   3.94363  -6.04943  -0.441188    -8.54497
 313 │ U.K.       1968   3.9532   -6.03038  -0.462451    -8.50187
 314 │ U.K.       1969   3.94806  -6.03195  -0.383325    -8.46812
 315 │ U.K.       1970   3.97971  -6.00826  -0.41899     -8.44514
 316 │ U.K.       1971   3.98365  -5.98173  -0.46136     -8.40279
 317 │ U.K.       1972   3.99276  -5.97301  -0.527772    -8.3561
 318 │ U.K.       1973   3.99536  -5.89756  -0.565297    -8.30668
 319 │ U.K.       1974   3.96696  -5.91281  -0.566413    -8.28577
 320 │ U.K.       1975   3.91258  -5.92324  -0.208674    -8.27642
 321 │ U.K.       1976   3.96231  -5.88681  -0.27354     -8.26937
 322 │ U.K.       1977   3.94555  -5.87101  -0.508863    -8.26652
 323 │ U.K.       1978   4.00037  -5.84055  -0.786529    -8.26261
 324 │ U.S.A.     1960   4.82396  -5.69837  -1.12111     -8.01946
 325 │ U.S.A.     1961   4.79632  -5.69522  -1.14624     -7.99926
 326 │ U.S.A.     1962   4.79894  -5.64881  -1.16187     -7.98641
 327 │ U.S.A.     1963   4.7879   -5.62685  -1.17992     -7.95954
 328 │ U.S.A.     1964   4.80827  -5.58715  -1.20026     -7.9299
 329 │ U.S.A.     1965   4.80595  -5.53689  -1.19429     -7.90049
 330 │ U.S.A.     1966   4.80753  -5.47901  -1.19026     -7.86798
 331 │ U.S.A.     1967   4.80088  -5.46496  -1.18991     -7.84118
 332 │ U.S.A.     1968   4.83414  -5.43021  -1.2073      -7.82243
 333 │ U.S.A.     1969   4.84138  -5.41437  -1.22314     -7.79271
 334 │ U.S.A.     1970   4.84848  -5.42196  -1.25176     -7.76538
 335 │ U.S.A.     1971   4.86029  -5.40604  -1.28132     -7.74892
 336 │ U.S.A.     1972   4.85407  -5.35396  -1.33117     -7.68042
 337 │ U.S.A.     1973   4.84637  -5.30816  -1.29067     -7.63769
 338 │ U.S.A.     1974   4.79863  -5.32869  -1.23147     -7.61756
 339 │ U.S.A.     1975   4.80493  -5.34619  -1.20038     -7.60701
 340 │ U.S.A.     1976   4.81489  -5.29795  -1.15468     -7.57475
 341 │ U.S.A.     1977   4.81103  -5.25661  -1.17591     -7.55346
 342 │ U.S.A.     1978   4.81845  -5.22123  -1.21206     -7.53618
